README.md

```diff
@@ -8,4 +8,4 @@ An open source project to practice client-server intergration
  *  Mongoose.js
  *  Faker.js
 
-### test push 2
\ No newline at end of file
+### test push 3
```

Comment:
We should remove this tests line, don't you think so? We don'T have to keep track manually there.

Reply:
removed. I was not aware of the one in the Backend project.